vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
But do we really need v1.0.0 , not master. Otherwise we can just comment https://github.com/rancher/os/blob/master/trash.conf#L32 if trash supports commenting, otherwise to just remove. Since I don't think v1.0.0 tag of go-shellwords repo will be ever updated IMO